chain-connect/docs/Examples.md

```diff
@@ -0,0 +1,319 @@
+            isConnected,
+        };
+    },
+};
+</script>
+```
+
+
+### Generating Wallets
+
+You can generate new wallets using the `WalletUtils` utility provided by GalaChain Connect.
+
+#### Creating a Random Wallet
+
+```typescript
+import { WalletUtils } from "@gala-chain/connect";
+
+// Generate a random wallet
+const randomWallet = WalletUtils.createRandom();
+
+console.log("GalaChain Address:", randomWallet.galachainAddress);
+console.log("Private Key:", randomWallet.privateKey);
+console.log("Public Key:", randomWallet.publicKey);
+console.log("Ethereum Address:", randomWallet.ethAddress);
+console.log("Mnemonic:", randomWallet.mnemonic?.phrase);
+```
+
+The `createRandom()` method returns an object containing:
+
+- `galachainAddress`: The GalaChain address of the wallet.
+- `privateKey`: The private key of the wallet.
+- `publicKey`: The public key of the wallet.
+- `ethAddress`: The Ethereum address of the wallet.
+- `mnemonic`: The mnemonic phrase used to generate the wallet (may be `null`).
+
+#### Creating and Registering a Random Wallet
+
+To create a random wallet and register it with a GalaChain service, use `createAndRegisterRandomWallet`:
+
+```typescript
+import { WalletUtils } from "@gala-chain/connect";
+
+// Replace with your registration endpoint, or use this to test stage
+const registrationEndpoint = "https://stage-galaswap.gala.com/v1/CreateHeadlessWallet";
+
+async function createAndRegisterWallet() {
+  try {
+    const randomWallet = await WalletUtils.createAndRegisterRandomWallet(registrationEndpoint);
+
+    console.log("Private Key:", randomWallet.privateKey);
+  } catch (error) {
+    console.error("Failed to create and register wallet:", error);
+  }
+}
+
+createAndRegisterWallet();
+```
+
+The `createAndRegisterRandomWallet()` function returns an object containing:
+
+- `galachainAddress`: The GalaChain address of the wallet.
+- `privateKey`: The private key of the wallet.
+- `publicKey`: The public key of the wallet.
+- `ethAddress`: The Ethereum address of the wallet.
+- `mnemonic`: The mnemonic phrase used to generate the wallet (may be `null`).
+
+**Example Usage in an Application:**
+
+```typescript
+import { WalletUtils } from "@gala-chain/connect";
+import { ref } from "vue";
+
+export const message = ref("");
+export const createdUser = ref<string | null>(null);
+
+export async function generateWallet() {
+  try {
+    const wallet = await WalletUtils.createAndRegisterRandomWallet(
+      "https://stage-galaswap.gala.com/v1/CreateHeadlessWallet"
+    );
+    message.value = `Wallet generated and registered. Address: ${wallet.ethAddress}`;
+    createdUser.value = wallet.ethAddress;
+
+    //This is unsafe, please don't use in production
+    console.log("Private Key:", wallet.privateKey);
+  } catch (error) {
+    message.value = "Failed to generate wallet!";
+    console.error(error);
+  }
+}
+```
+
+### Template Integration
+
+In your Vue component, you can add a button to generate a wallet:
```

Comment:
Same thing here, remove examples related to html implementation using vue

Reply:
Removed!